src/plugins/vis_builder/opensearch_dashboards.json

```diff
@@ -1,5 +1,6 @@
 {
-  "id": "wizard",
+  "id": "visBuilder",
+  "configPath": "visBuilder",
```

Comment:
Add a `configPath: 'visBuilder'` here if we want to use `visBuilder.enabled: false` in the yml file.

Otherwise, because we change the id from wizard to visBuilder without specifying a configPath, then the yml file will just take the value of the id, and go through some underlying mechanisms, and the config has to be: `vis_builder: false` in the yml file in order for it to work. @ashwin-pc @joshuarrrr

Reply:
I was just going to ask about this. I'm actually okay with it being `vis_builder:false`.

Reply:
cool, i can get rid of the configPath and change the config to vis_builder:false